_posts/2020-02-26-#17428.md

```diff
@@ -51,87 +52,96 @@ startup.
   restart.
 
 ### Issue 17326
+
 - The most likely way for an eclipse attack to occur is by forcing a victim
-  to restart and then taking over its connections with malicious peers.
+  to restart and then making it connect only to malicious peers.
 
-- In the 2015 implementation of Bitcoin, an attacker could carry out a
-  "connection starvation attack" where the attacker forces a victim to reboot and fills up all the free
-  connection slots in the network, disabling the victim from connecting to any
-  nodes. So, anchor connections would not have been an effective mitigation
-  strategy.
+- In the 2015 implementation of Bitcoin Core, an attacker could carry out a
+  "connection starvation attack". In this attack, the adversary fills up
+  all of the available inbound connections on the network and then forces
+  the victim to reboot. When the victim restarts, he's unable to connect to
+  any honest peers on the network since their inbound connections slots are
+  all filled up. Anchor connections are not an effective mitigation
+  strategy against such an attack.
```

Comment:
I think we can elaborate slightly further...

Maybe...
Since anchor connections only help identify honest peers, not connect to them, they are not an effective mitigation strategy against such an attack.

Reply:
good change. Fixed!